src/eclipse/eclipse.importorder

```diff
@@ -0,0 +1,7 @@
+#Organize Import Order
+#Wed Apr 26 12:53:22 EDT 2017
+4=\#
+3=org.springframework
+2=
+1=javax
+0=java
```

Comment:
A-ha good find! Yeah, I've got that disabled. Will add a new line to `CONTRIBUTING.adoc`.